tests/units/fastsync/test_fastsync_tap_s3_csv.py

```diff
@@ -36,10 +36,10 @@ def setUp(self) -> None:
 
         def tap_type_to_target_type(tap_type):
             return {
-                'Bool': 'smallint',
-                'Integer': 'number',
-                'Decimal': 'number',
-                'String': 'varchar'
+                'boolean': 'boolean',
```

Comment:
Should this not be varchar also ?

Reply:
That's just a mocked function. it can be anything you want. It's injected when creating a class instance.